public/nmsce.js

```diff
@@ -1487,7 +1513,7 @@ class NMSCE {
     }
 
     openSearch() {
-        window.open("this.html?s=" + this.last.addr.nameToId() + "&g=" + this.last.galaxy.nameToId(), '_self')
+        window.open("bhs?s=" + this.last.addr.nameToId() + "&g=" + this.last.galaxy.nameToId(), '_self')
```

Comment:
This is not correct. neither `this.html` nor `bhs` will resolve. Has to be changed before it gets merged